src/dev/dev.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Controller, Get, Inject, QueryParams, UseBefore } from '@tsed/common';
+import { ContentType, Description, Hidden } from '@tsed/schema';
+
+import DevelopmentFilter from '../common/filters/development-filter';
+import { DevelopmentService } from './dev.service';
+
+@Controller('/dev')
+@UseBefore(DevelopmentFilter)
+@Hidden()
+export class DevelopmentController {
+  @Inject()
+  developmentService!: DevelopmentService;
+
+  @Get('/test')
+  @ContentType('json')
+  @Description('Put any function that u want to test out')
+  async test(@QueryParams('param') param: string): Promise<string> {
```

Comment:
remove this one or wants to keep it?

Reply:
ye, i wonnna keep this for custom launches